chart/cluster-proxy-addon/templates/clustermanagementaddon.yaml

```diff
@@ -6,6 +6,8 @@ spec:
   addOnMeta:
     displayName: cluster-proxy
     description: cluster-proxy
-  addOnConfiguration:
-    crdName: managedproxyconfigurations.proxy.open-cluster-management.io
-    crName: cluster-proxy
+  supportedConfigs:
```

Comment:
@skeeey Here we have to change to supportedConfigs otherwise an error will ocurr.

Reply:
error manage proxy configurations: []

Reply:
The error comes from the addon-deploy-controller

Reply:
This is  our expected,  the `addOnConfiguration` has been deprecated, here we use the new way to declare the default/global config for cluster-proxy

Reply:
Got it.